packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -347,6 +347,11 @@ IEventProvider<IFluidDataStoreContextEvents>, Partial<IProvideFluidDataStoreRegi
      * and its children with the GC details from the previous summary.
      */
     getInitialGCSummaryDetails(): Promise<IGarbageCollectionSummaryDetails>;
+
+    /**
+     * Tells whether the data store is referenced as per the initial summary when it was loaded.
+     */
+    isReferencedInInitialSummary(): Promise<boolean>;
```

Comment:
do we need this on the interface? can we get by with this just on the implementation?

Reply:
i really don't think this is the right design, so would like to minimize this as much as possible